.github/workflows/push-to-azure-queue.yml

```diff
@@ -35,8 +36,18 @@ jobs:
     runs-on: ubuntu-latest
     needs: [ queue-pr ]
     steps:
-      - name: Create commit comment
-        uses: peter-evans/commit-comment@3f3629d9da4bf387b09fc75456a5b8a344ba5641 # pin@v2
+      - name: Create Comment on PR
+        uses: actions/github-script@7a5c598405937d486b0331594b5da2b14db670da # pin@v6
         with:
-          body: |
-            'This commit is queued for merging with the `test_next` branch! Please ignore this PR for now. Contact @microsoft/fluid-cr-infra for help.'
+          token: ${{ secrets.WORKFLOW_TOKEN }}
+          script: |
+            const { Octokit } = require("@octokit/core");
+            const pr = ${{ github.event.pull_request.number }}
+            const token = `${{ secrets.GITHUB_TOKEN }}`;
+            const octokit = new Octokit({ auth: token })
+            octokit.rest.pulls.createReviewComment({
+              owner: 'sonalivdeshpande',
+              repo: 'FluidFramework',
+              pull_number: pr,
+              body: 'This commit is queued for merging with the `test_next` branch! Please ignore this PR for now. Contact @microsoft/fluid-cr-infra for help.',
+            });
```

Comment:
This is OK, but consider using [this action](https://github.com/marocchino/sticky-pull-request-comment) that we already use elsewhere. It's quite nice.

https://github.com/microsoft/FluidFramework/blob/11848cd181cad0729b656cf9e3e5a9fe11816942/.github/workflows/linkcheck-reporter.yml#L24-L30

That example uses a file as input, but there are other simple examples in the action docs.